pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-stop.adoc

```diff
@@ -0,0 +1,49 @@
+= Stopping Request Execution
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+To intercept and stop a request, return a `Response` object from your `on_request` wrapped functions:
+
+[source,Rust]
+----
+Response::new(401)
+    .with_headers(vec![("WWW-Authenticate".to_string(), "Bearer realm=\"oauth2\"".to_string())])
+    .with_body(r#"{ "error": "token was not present"}"#)
+----
+
+You can block or allow requests to reach the upstream service using a `Flow`. A `Flow` in an `enum` value with two possible values:
+
+* `Continue`: Lets the request pass. 
++
+This element defines an object that is forwarded to the response.
```

Comment:
Is this the same as "Let's the request pass"? If so I'm thinking this is redundant.